jiant/scripts/postproc/xtreme/README.md

```diff
@@ -0,0 +1,369 @@
+    --output_dir ${BASE_PATH}/cache/${MODEL_TYPE}/${TASK} \
+    --phases train,val \
+    --max_seq_length 384 \
+    --smart_truncate
+for LG in ar bn fi id ko ru sw te; do
+    TASK=tydiqa_${LG}
+    python jiant/proj/main/tokenize_and_cache.py \
+        --task_config_path ${BASE_PATH}/tasks/configs/${TASK}_config.json \
+        --model_type ${MODEL_TYPE} \
+        --model_tokenizer_path ${BASE_PATH}/models/${MODEL_TYPE}/tokenizer \
+        --output_dir ${BASE_PATH}/cache/${MODEL_TYPE}/${TASK} \
+        --phases val \
+        --max_seq_length 384 \
+        --smart_truncate
+done
+```
+
+### Bucc2018
+```bash
+for LG in de fr ru zh; do
+    TASK=bucc2018_${LG}
+    python jiant/proj/main/tokenize_and_cache.py \
+        --task_config_path ${BASE_PATH}/tasks/configs/${TASK}_config.json \
+        --model_type ${MODEL_TYPE} \
+        --model_tokenizer_path ${BASE_PATH}/models/${MODEL_TYPE}/tokenizer \
+        --output_dir ${BASE_PATH}/cache/${MODEL_TYPE}/${TASK} \
+        --phases val,test \
+        --max_seq_length 512 \
+        --smart_truncate
+done
+```
+
+### Tatoeba
+```bash
+for LG in af ar bg bn de el es et eu fa fi fr he hi hu id it ja jv ka kk ko ml mr nl pt ru sw ta te th tl tr ur vi zh; do
+    TASK=tatoeba_${LG}
+    python jiant/proj/main/tokenize_and_cache.py \
+        --task_config_path ${BASE_PATH}/tasks/configs/${TASK}_config.json \
+        --model_type ${MODEL_TYPE} \
+        --model_tokenizer_path ${BASE_PATH}/models/${MODEL_TYPE}/tokenizer \
+        --output_dir ${BASE_PATH}/cache/${MODEL_TYPE}/${TASK} \
+        --phases val \
+        --max_seq_length 512 \
+        --smart_truncate
+done
+```
+
+### MNLI and SQuAD
+```bash
+TASK=mnli
+python jiant/proj/main/tokenize_and_cache.py \
+    --task_config_path ${BASE_PATH}/tasks/configs/${TASK}_config.json \
+    --model_type ${MODEL_TYPE} \
+    --model_tokenizer_path ${BASE_PATH}/models/${MODEL_TYPE}/tokenizer \
+    --output_dir ${BASE_PATH}/cache/${MODEL_TYPE}/${TASK} \
+    --phases train,val \
+    --max_seq_length 256 \
+    --smart_truncate
+TASK=squad_v1
+python jiant/proj/main/tokenize_and_cache.py \
+    --task_config_path ${BASE_PATH}/tasks/configs/${TASK}_config.json \
+    --model_type ${MODEL_TYPE} \
+    --model_tokenizer_path ${BASE_PATH}/models/${MODEL_TYPE}/tokenizer \
+    --output_dir ${BASE_PATH}/cache/${MODEL_TYPE}/${TASK} \
+    --phases train,val \
+    --max_seq_length 384 \
+    --smart_truncate
+```
+
+## Generate Run configs
+
+Now, we generate the run configurations for each of our XTREME tasks. Each of the 9 XTREME tasks will correspond to one run. It's worth noting here:
+
+* XNLI uses MNLI for training, XQuAD and MLQA use SQuAD v1.1, while PAWS-X, UDPOS, PANX and TyDiQA have their own English training sets.
+* For these tasks, we will train on the training set, and then evaluate the validation set for all available languages.
+* Bucc2018 and Tatoeba, the sentence retrieval tasks, are not trained, and only run in evaluation mode.
```

Comment:
Why? Is there another source of info to understand the methodology here?

Reply:
This is something that's built into the benchmark, so any user of it should be familiar with the justification. I will add a link to the paper/reference implementation though.